test/test.js

```diff
@@ -918,6 +953,32 @@ test('don\'t display test title, if there is only one anonymous test', function
 	});
 });
 
+test('don\'t display hook title, if it did not fail ', function (t) {
+	t.plan(2);
+
+	fork(path.join(__dirname, 'fixture', 'hooks-passing.js'))
+		.on('test', function (test) {
+			t.deepEqual(test.err, {});
+			t.is(test.title, 'pass');
+		})
+		.then(function () {
+			t.end();
+		});
```

Comment:
Just tried it, does not work, because `fork` resolves with test results: `t.end(results)`.